resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -163,6 +168,18 @@ public Builder recordFailure(Predicate<? super Throwable> predicate) {
             return this;
         }
 
+        /**
+         * Configures whether a circuit breaker can automatically transition from OPEN to HALF_OPEN state once the waitDurationInOpenState
+         * has passed. Defaults to false.
+         *
+         * @param enableAutomaticTransitionFromOpenToHalfOpen true to enable.
+         * @return the CircuitBreakerConfig.Builder
+         */
+        public Builder enableAutomaticTransitionFromOpenToHalfOpen(boolean enableAutomaticTransitionFromOpenToHalfOpen) {
+            config.enableAutomaticTransitionFromOpenToHalfOpen = enableAutomaticTransitionFromOpenToHalfOpen;
```

Comment:
Let's remove the parameter `enableAutomaticTransitionFromOpenToHalfOpen`.
The method could always set `enableAutomaticTransitionFromOpenToHalfOpen` to true.
Since a CircuitBreakerConfig is immutable you can never change it back to false anyway.